snapcraft/_baseplugin.py

```diff
@@ -75,11 +75,6 @@ def schema(cls):
             'build-properties': ['disable-parallel']
         }
 
-    @property
-    def PLUGIN_STAGE_SOURCES(self):
```

Comment:
we just had a talk about breaking API :smile:

instead of removing this and introducing `get_stage_sources` why don't you just override this property?